<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="simulatorControlsGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>321</width>
      <height>921</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulator Controls</string>
    </property>
    <widget class="QGroupBox" name="playbackControlsGroup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>261</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Playback</string>
     </property>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="CGMControlsGroup">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>CGM Reading</string>
     </property>
     <widget class="QDoubleSpinBox" name="BGdiffValue">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>18.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>5.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="BGdiffLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Blood Sugar (mmol/L)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="applyBGdiffButton">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>70</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Device Components</string>
     </property>
     <widget class="QPushButton" name="killBatteryButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill Battery</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chargeBatteryButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Charge Battery</string>
      </property>
     </widget>
     <widget class="QPushButton" name="emptyCartridgeButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Empty Cartridge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="replaceCartridgeButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Replace Cartridge</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="deviceDisplayWidget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>1200</width>
      <height>900</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
